src/App.js

```diff
@@ -346,6 +346,10 @@ function App() {
   } else {
     return (
       <div className="fade">
+<<<<<<< HEAD
+=======
+        <Header />
+>>>>>>> main
```

Comment:
Remove this line finally then save the file.

Reply:
thanks, I have made the changes kindly check my new commit for the latest updates @shehabadel